src/ParallelHashMap.h

```diff
@@ -273,8 +270,8 @@ void FixedHashMap<K,V>::insert(K key, V value)
  *      key was already present in map.
  */
 template <class K, class V>
-int FixedHashMap<K,V>::insert_and_get_count(K key, V value)
-{
+int FixedHashMap<K,V>::insert_and_get_count(K key, V value) {
+
   /* get hash into table using fast modulus */
   size_t key_hash = std::hash<K>()(key) & (_M-1);
 
```

Comment:
Can you remove the braces around the critical statement on line 298. Also what effect of the `(node_incr)` after critical? I've never seen a variable in parentheses after critical before and am more curious what that does.